src/binary_tree/bst.c

```diff
@@ -0,0 +1,139 @@
+#include "./binary_tree.h"
+#include <stdlib.h>
+
+dsBinaryTree_t *dsBSTCreateNode(void *value) 
```

Comment:
Web should use the `static` keyword when It shouldn't be accessed outside the file.